### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christianbmadsen**
Adding link to tensor-network-ahs

*Issue #, if available:* None

*Description of changes:* Adding link to the new tensor-network-ahs repo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ajberdy** [comment]
Screenshot 2023-12-08 at 1 18 13 PM
render doesn't look quite right

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ We welcome community contributions for Amazon Braket and are excited to highligh
 * ![Python](https://img.shields.io/badge/python-blue?logo=python&logoColor=ffdd54) [Bloqade](https://github.com/QuEraComputing/bloqade-python): Bloqade is an SDK designed to make writing and analyzing the results of analog quantum programs on QuEra's neutral atom quantum computers as seamless and flexible as possible.
 * ![Python](https://img.shields.io/badge/python-blue?logo=python&logoColor=ffdd54) [OpenQAOA](https://github.com/entropicalabs/openqaoa): A python library from EntropicaLabs for quantum optimization using QAOA on Quantum computers and Quantum computer simulators.
 * ![Python](https://img.shields.io/badge/python-blue?logo=python&logoColor=ffdd54) [Quantum state preparation](https://github.com/guikaiwen/qubit_efficient_QSP): Implementations and performance testings for quantum state preparation circuits.
+* ![Julia](https://github.com/amazon-braket/tensor-network-ahs) [Tensor Network for Analog Hamiltonian Simulation]: This package implements a tensor network based algorithm for simulating Rydberg atom dynamics.
```
Comment (ajberdy) [inline README.md:15]: Looks like this url should be the julia logo and then this link should be after the TN for AHS title

Comment (ajberdy) [reply README.md:15]:
```suggestion
* ![Julia](https://img.shields.io/badge/julia-white?logo=julia) [Tensor Network for Analog Hamiltonian Simulation](https://github.com/amazon-braket/tensor-network-ahs): This package implements a tensor network based algorithm for simulating Rydberg atom dynamics.
```

**Comment by zmohammad01** [comment]
Seems like the readme is updated with the right information now - feel free to just close this :)

**Review by ajberdy — APPROVED** [review]

**Comment by ajberdy** [comment]
> Seems like the readme is updated with the right information now - feel free to just close this :)

I don't see the link for the Julia ahs sim on main. What are you referring to?

**Pull request merged**